src/System.Runtime.Intrinsics.Experimental/ref/System.Runtime.Intrinsics.cs

```diff
@@ -25,6 +25,177 @@ public struct Vector256<T> where T : struct
          private int _dummy;
     }
 }
+namespace System.Runtime.Intrinsics.Arm.Arm64
+{
+    public static class Aes
+    {
+        public static bool IsSupported { get { throw null; } }
+        public static Vector128<byte> Decrypt(Vector128<byte> value, Vector128<byte> roundKey) { throw null(); }
```

Comment:
`null()` doesn't compile.

Reply:
I had regular expressions enabled when I replaced 
`thow new throw new PlatformNotSupportedException()` with `throw null` so the unescaped `()` was ignored.